### PR TITLE
Client changes and bug fixes

### PR DESCRIPTION
**Pull request by priyankasprakash**

### CODE REVIEW

File: Node-DC-EIS-client/node_dc_eis_testurls.py [hunk 1]
```diff
@@ -191,16 +202,31 @@ def post_url(url,request_num,log,phase):
...
     post_data = post_datalist[0]
+  else:
+    static_post = static_post + 1
   start = time.time()
   end = start
   for i in range(100):
     r =  post_function(url, post_data)
     if r:
       end = time.time()
+      content_len = len(r.content)
```
Comment (egalli) [inline Node-DC-EIS-client/node_dc_eis_testurls.py]: please extract the length calculation into a separate function and call it from the post and get

File: Node-DC-EIS-client/node_dc_eis_testurls.py [hunk 2]
```diff
@@ -320,8 +359,8 @@ def main_entry(url, request_num, url_type, log_dir, phase, interval,
...
   if url_type == 1:
-    get_url(url, request_num, log, phase, accept_header)
+    get_url(url,url_type,request_num, log, phase, accept_header)
   if url_type == 2:
-    post_url(url,request_num,log,phase)
+    post_url(url,url_type,request_num,log,phase)
   if url_type == 3:
-    delete_url(url,request_num,log,phase)
\ No newline at end of file
+    delete_url(url,url_type,request_num,log,phase)
```
Comment (egalli) [inline Node-DC-EIS-client/node_dc_eis_testurls.py]: the github seems to be complaining about the lack of new line at the end. Should we add one?

**Review by egalli — CHANGES_REQUESTED** [review]

**Comment by uttampawar** [comment]
@priyankasprakash Any update on the fix?

File: Node-DC-EIS-client/node_dc_eis_testurls.py (hunk 2 above)
Comment (priyankasprakash) [reply Node-DC-EIS-client/node_dc_eis_testurls.py]: It complains even if I add a new line. Any idea how to fix this?

**Review by egalli — APPROVED** [review]

**Pull request merged**